.codegen/error_mapping.py.tmpl

```diff
@@ -0,0 +1,49 @@
+# Code generated from OpenAPI specs by Databricks SDK Generator. DO NOT EDIT.
+
+from .base import DatabricksError
+
+__all__ = ['error_mapper'{{range .ExceptionTypes}}, '{{.PascalName}}'{{end}}]
```

Comment:
Is this necessary? These are also the only exported members of this module. I also wasn't aware that this variable was respected in modules that also define an `__init__.py`

Reply:
this is how python works ;)

Reply:
I just cloned this PR and removed this line and was still able to import these:
```py
from databricks.sdk import errors

print(dir(errors))
```
```
['Aborted', 'AlreadyExists', 'BadRequest', 'Cancelled', 'DataLoss', 'DatabricksError', 'DeadlineExceeded', 'ErrorDetail', 'InternalError', 'InvalidParameterValue', 'NotFound', 'NotImplemented', 'OperationFailed', 'OperationTimeout', 'PermissionDenied', 'RequestLimitExceeded', 'ResourceAlreadyExists', 'ResourceConflict', 'ResourceExhausted', 'TemporarilyUnavailable', 'TooManyRequests', 'Unauthenticated', 'Unknown', '__builtins__', '__cached__', '__doc__', '__file__', '__loader__', '__name__', '__package__', '__path__', '__spec__', 'base', 'error_mapper', 'mapper', 'mapping', 'sdk']
```
If you want to control what is exported, I think you need to include this in the `__init__.py` file explicitly.